packages/sources/avalanche-platform/src/index.ts

```diff
@@ -1,10 +1,13 @@
-import { expose } from '@chainlink/ea-bootstrap'
-import { makeExecute, endpointSelector } from './adapter'
-import * as endpoints from './endpoint'
-import { makeConfig, NAME } from './config'
-import { envDefaultOverrides } from './config/envDefaultOverrides'
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { PoRAdapter } from '@chainlink/external-adapter-framework/adapter/por'
+import { config } from './config'
+import { balance } from './endpoint'
 
-const adapterContext = { name: NAME, envDefaultOverrides }
+export const adapter = new PoRAdapter({
+  defaultEndpoint: balance.name,
+  name: 'AVALANCHE_PLATFORM',
+  config,
+  endpoints: [balance],
+})
```

Comment:
Since this adapter uses an HttpTransport, we'll want to add a reasonable limit here. I believe this RPC call is a heavier call on the node so we can set it conservatively at 10 req/s.
```suggestion
export const adapter = new PoRAdapter({
  defaultEndpoint: balance.name,
  name: 'AVALANCHE_PLATFORM',
  config,
  endpoints: [balance],
  rateLimiting: {
    tiers: {
      default: {
        rateLimit1s: 10
      },
    },
  },
})
```